### DISCUSSION TITLE
not an issue: testkey pass and alias

### DISCUSSION THREAD

**Discussion by STICKnoLOGIC** [Sketchware-Pro/Sketchware-Pro#717] [Q&A] [answered]
Hi! can anyone tell me what is the password/alias/alias-pass of the testkey.keystore here in the root directory of the repository
![Screenshot_20230118_163003](https://user-images.githubusercontent.com/65322242/213121663-9fb19a03-cb78-4270-8cb1-6a26688ed6b0.jpg)

**Comment by j-px** [comment, marked as answer]
See here:

https://github.com/Sketchware-Pro/Sketchware-Pro/blob/bb473fe5ed3ea81c5e3a40d0a56061f47734056b/app/build.gradle#L81-L88

**Comment by STICKnoLOGIC** [comment]
many thanks
